public/styles/dark.css

```diff
@@ -47,7 +47,7 @@
     --color-decorative-4: var(--high-violet);
     --color-decorative-5: var(--deep-sea);
 
-    --color-doc-cover: var(--gravel-rocks);
+    --color-doc-cover: var(--sea-anchor);
```

Comment:
Same here - you must verify this change doesn't break any of its usages.

Reply:
Checked also this one on most of the places, it seems cool, a bit darker color.